src/components/UniversalNavbar/UniversalNavbar.js

```diff
@@ -10,211 +9,157 @@ import { Button, Layout, Spacer, TitleXLarge } from '../index'
 import TransformingBurgerButton from './TransformingBurgerButton/TransformingBurgerButton'
 import NavLink from './NavLink'
 
+import { LINKS } from './constants'
 import styles from './UniversalNavbar.module.scss'
 
-// TODO REDESIGN: Lots of sloppy inline styles here.
-
-const LINKS = {
-  // These are used e.g. in the logo and CTA button:
-  INDEX: { href: '/' },
-  TERM: { href: '/term' },
-
-  // These are used in the navigation links proper:
-  NAVLINKS: [
-    {
-      id: uuidv4(),
-      href: '/how-it-works/',
-      title: 'How it works',
-    },
-    {
-      id: uuidv4(),
-      href: '/why-ethos/',
-      title: 'Why Ethos',
-    },
-    {
-      id: uuidv4(),
-      href: '/faq/',
-      title: 'FAQ',
-    },
-    {
-      id: uuidv4(),
-      href: '/search/',
-      title: 'Search',
-    },
-    {
-      id: uuidv4(),
-      href: '/login/',
-      title: 'Account',
-    },
-  ],
-}
-
-class UniversalNavbar extends React.Component {
-  state = {
-    showMobileMenu: false,
-  }
-
-  toggleHamburger = () => {
-    // TODO: use functional setState
-    this.setState({ showMobileMenu: !this.state.showMobileMenu })
+const UniversalNavbar = ({
+  LinkComponent,
+  hideMobileCta,
+  hideDesktopCta,
+  logoHref,
+  trackCtaClick,
+}) => {
+  const [showMobileMenu, setShowMobileMenu] = useState(false)
+
+  const toggleHamburger = () => {
+    setShowMobileMenu(!showMobileMenu)
   }
 
-  handleHamburgerKeyPress = (event) => {
+  const handleHamburgerKeyPress = (event) => {
     if (event.key === 'Enter') {
-      this.toggleHamburger()
+      toggleHamburger()
     }
   }
 
-  render() {
-    const {
-      LinkComponent,
-      hideMobileCta,
-      hideDesktopCta,
-      logoHref,
-    } = this.props
-
-    const renderCtaButton = (showWhenScrolled) => (
-      <a
-        className={
-          'cta-button ' + (showWhenScrolled ? 'show-when-scrolled' : '')
-        }
-        onClick={this.props.trackCtaClick}
-        href={LINKS.TERM.href}
+  const renderCtaButton = (showWhenScrolled) => (
+    <a
+      className={'cta-button ' + (showWhenScrolled ? 'show-when-scrolled' : '')}
```

Comment:
Can these use CSS Modules?

Reply:
This is still relying on legacy styles from `FancyAnimatedLogo.scss`. We can move this to CSS Modules as a followup and fix.